py-tidal/setup.py

```diff
@@ -0,0 +1,28 @@
+import re
+import os
+import codecs
+from setuptools import setup, find_packages
+
+
+_base_dir = os.path.dirname(__file__)
+
+package_name = "py-tidal".replace("_", "-").lower()
+
+def get_requirements():
+    with open(os.path.join(_base_dir, "requirements.txt")) as fp:
+        return fp.read().splitlines()
+
+setup(
+    name = package_name,
+    version = "0.0.1",
+    description = "Python Tidal port",
+    author = "Tydal Ciclists",
+    author_email = "artheist@lgmail.com",
```

Comment:
I think we can put a list of emails here, or we could make an alias under `@tidalcycles.org` that goes to multiple addresses

Reply:
yeah, I was thinking about it, an alias would be best

Reply:
I took vortex@tidalcycles.org hoping that's suitable ...